pkgs/top-level/all-packages.nix

```diff
@@ -11943,7 +11943,7 @@ in
 
   amsn = callPackage ../applications/networking/instant-messengers/amsn { };
 
-  antimony = qt5.callPackage ../applications/graphics/antimony {};
+  antimony = qt55.callPackage ../applications/graphics/antimony {};
```

Comment:
Thanks for looking into this @abbradar 

I saw your comment regarding patching `nixos/modules/config/system-path.nix` but I'm not sure how to do that yet.

I'm still trying to track down QT apps I have to get rid of in my configuration.nix. I also did an `ldd /run/current-system/sw/bin/*` but couldn't find anything to disable.

Any pointers?